### PR TITLE
Fix memory allocate for SVM's

### PR DESCRIPTION
**Pull request by PetrovKP**

### CODE REVIEW

File: cpp/daal/src/algorithms/svm/svm_predict_impl.i [hunk 1]
```diff
@@ -204,9 +204,8 @@ struct SVMPredictImpl<defaultDense, algorithmFPType, cpu> : public Kernel
         const size_t nVectors = xTable->getNumberOfRows();
         const size_t nSV      = svTable->getNumberOfRows();
 
-        size_t nRowsPerBlock = 0;
-        DAAL_SAFE_CPU_CALL((nRowsPerBlock = 128), (nRowsPerBlock = nVectors));
-        const size_t nBlocks = nVectors / nRowsPerBlock + !!(nVectors % nRowsPerBlock);
+        const size_t nRowsPerBlock = 128;
+        const size_t nBlocks       = nVectors / nRowsPerBlock + !!(nVectors % nRowsPerBlock);
```
Comment (KulikovNikita) [inline cpp/daal/src/algorithms/svm/svm_predict_impl.i:208]:
```suggestion
        const size_t nBlocks       = nVectors / nRowsPerBlock + bool(nVectors % nRowsPerBlock);
```

File: cpp/daal/src/algorithms/svm/svm_predict_impl.i [hunk 2]
```diff
@@ -204,9 +204,8 @@ struct SVMPredictImpl<defaultDense, algorithmFPType, cpu> : public Kernel
         const size_t nVectors = xTable->getNumberOfRows();
         const size_t nSV      = svTable->getNumberOfRows();
 
-        size_t nRowsPerBlock = 0;
-        DAAL_SAFE_CPU_CALL((nRowsPerBlock = 128), (nRowsPerBlock = nVectors));
-        const size_t nBlocks = nVectors / nRowsPerBlock + !!(nVectors % nRowsPerBlock);
+        const size_t nRowsPerBlock = 128;
```
Comment (KulikovNikita) [inline cpp/daal/src/algorithms/svm/svm_predict_impl.i:207]: Should it be fixed? Are any empiricisms that can help to get this constant based on HW properties & data size?

**Review by KulikovNikita — APPROVED** [review]

File: cpp/daal/src/algorithms/svm/svm_predict_impl.i (hunk 2 above)
Comment (PetrovKP) [reply cpp/daal/src/algorithms/svm/svm_predict_impl.i:207]: Yes, we can. In the future it is planned

**Review by owerbat — APPROVED** [review]

**Pull request merged**